README.md

```diff
@@ -53,168 +56,30 @@ If a direct File Share transfer is used, there is effectively no limit to the pa
-* Left-click the `Show All Files` button within the toolbar at the top of the Solution Explorer
-* Hold the `SHIFT` key and left-click all copied files
-* Left-click the `Include In Project` menu item
-
-The window should look like this:
-
-![includeinproject](https://user-images.githubusercontent.com/32276163/45234663-486b7400-b28b-11e8-8bc9-f6b2dea7009f.png)
-
-Verify the solution builds successfully.
-
-#### Update file properties
-When adding existing files to a project, the `Copy to Output Directory` File property is set to `Do not copy` by default. Update all of the specified File properties to ensure those files are copied to the build directory.
-
-* Hold the `SHIFT` key and left-click all files within the `Resources` folder and `LogConfig.xml` (6 files)
-* Within the Properties window, change the `Copy to Output Directory` File property to `Copy always`
-* The File property is globally updated for all selected files
-
-The window should look like this:
-
-![copyalways](https://user-images.githubusercontent.com/32276163/45234916-0a228480-b28c-11e8-82ad-ce22351e9dbf.png)
-
-
-Verify the solution builds successfully and the required files are copied correctly (assume **debug** build)
-
-```
-|-- bin
-    |-- Debug
-        |-- Resources
-            |-- EDRM-Sample1.jpg
-            |-- EDRM-Sample2.pdf
-            |-- EDRM-Sample3.doc
-            |-- EDRM-Sample4.xlsx
-            |-- EDRM-Sample5.pptx
-        |-- LogConfig.xml
-```
-
-#### Update App.config
-The [Json.NET](https://www.newtonsoft.com/json) is a popular JSON library that has been heavily used for many years. In fact, this library is so popular that [Microsoft's own documentation](https://docs.microsoft.com/en-us/dotnet/api/system.web.script.serialization.javascriptserializer?view=netframework-4.7.2) recommends using it for serialization and deserialization functionality. Because this library is prone to severe compatibility issues, Relativity strongly recommends defining a [binding redirect for Json.NET](#binding-redirect-for-json.net).
-
-Copy and paste the source below into the `App.config` file.
-
-<details><summary>View source code</summary>
+Verify the solution builds successfully. Even though the application performs no real work yet, debug to ensure the application terminates with a zero exit code.
 
-```xml
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
-    </startup>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-      <dependentAssembly>
-        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="6.0.0.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
-</configuration>
-```
+Ensure the following 5 settings found in app.config are updated with valid parameters:
+* TransferMode

```

Comment:
Please add a note that it is either `Aspera` or `Direct`.

Reply:
Done